### DISCUSSION TITLE
/clear doesnt clear armor

### DISCUSSION THREAD

**Discussion by Chris6ix** [EssentialsX/Essentials#5029] [Ideas and Feature Suggestions] [answered]
Hey :)
I'm not sure if this is a bug or if it's intended, but using `/clear` doesn't clear the armor from a player...
If this is a bug, then I can open an issue for that if you guys want to.
If it's intended, then I would like to suggest changing that because it is quite annoying when you want to clear a player, that you have to tell them each time that they have to unequip their armor first...

**Comment by Chew** [comment]
I believe you do `/clear all`

**Reply by Chris6ix** [reply]
No, that doesn't work, because the 1st argument of `/clear` is the targeted player...so doing `/clear all` tries to clear the inventory of the Player "all" which just fails with "Error: Player not found."

**Comment by pop4959** [comment, marked as answer]
You clear inventory items and armor with `/clear <username> **`.

`*` is inventory, `**` is inventory and armor.

Just be sure not to use `*` or `**` as the target selector (specify the username) as otherwise you will clear items from all players instead.

For example `/clear * **` to clear all inventory items and armor from all players.

**Reply by Chris6ix** [reply]
Ah, perfect!
That's exactly what I was looking for!
Thanks man :)
